commands/audit/scarunner.go

```diff
@@ -37,7 +37,19 @@ import (
 	xrayCmdUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
 )
 
-func buildDepTreeAndRunScaScan(auditParallelRunner *utils.SecurityParallelRunner, auditParams *AuditParams, results *xrayutils.Results) (err error) {
+func shouldPreformScaScan(cmdResults *results.SecurityCommandResults) bool {
+	if len(cmdResults.Targets) == 0 {
+		return false
+	}
+	for _, scan := range cmdResults.Targets {
```

Comment:
Can you explain this condition (to me + add it in a comment above) - it is not clear why this is a condition for executing or not executing sca scan...
Also from this condition we can infer that if one of the targets doesn't have a technology specified we run the SCA scan on all targets (since we return true) is that what you meant? why?